keyboards/fancytech/fancyalice66/config.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 chent7 (@chent7)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* RGB Matrix config */
+#define RGB_DI_PIN B0
```

Comment:
This can go to info.json (`rgblight.pin`, moving to `ws2812.pin` on `develop`).

Reply:
Moved to `info.json`